kubernetes_deploy/staging/queue_worker_deployment.yml

```diff
@@ -6,10 +6,16 @@ metadata:
   name: laa-legal-adviser-api-worker
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: laa-legal-adviser-api
+      tier: worker
+      env: staging
```

Comment:
_not a blocker, just voicing my thoughts_
💭 It felt natural to use the `env` as a selector but seeing this I'm wondering if it's enough to have `app` and `tier`, as we have namespace separation for environments already. My first instinct is to leave this as-is and come back to it if it proves too much extra labelling.

Reply:
I've got to admit, I don't like the `env` in there at all either